apps/fabric-website/src/pages/GetStarted/GetStartedPage.tsx

```diff
@@ -226,25 +231,29 @@ ReactDOM.render(<MyPage />, document.body.firstChild);`}
               </CodeBlock>
               <DefaultButton text="See Button" primary={true} href="#/components/button" />
               <p>
-                For more information about using components, check out the{' '}
+                Check out the
                 <a className={styles.getStartedLink} href="#/components">
                   components page
                 </a>
-                .
+                for more design and technical docs for each component.
               </p>
             </li>
             <li>
               <p>
                 If you are using Fabric React components that have icons, you can make all icons available by calling the{' '}
-                <code>initializeIcons</code> function from the <code>@uifabric/icons</code> package.
+                <code>initializeIcons</code> function from the{' '}
+                <a href="https://www.npmjs.com/package/@uifabric/icons">
```

Comment:
I'd also consider opening this link in a new tab so folks don't lose their place while reading this page.

Reply:
👍